tests/gui/builder/control/test_progress.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021-2024 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import taipy.gui.builder as tgb
+from taipy.gui import Gui
+
+
+def test_progress_builder(gui: Gui, helpers):
```

Comment:
add test with properties

Reply:
Which properties?

Reply:
liner, show_value, value

Reply:
Actually, what I'm trying to say that, even without the properties, the test is failing. It's showing assertion error that the `taipy.gui.builder` has no element `progress`?

Reply:
I know that's strange but might be sure to the fact that the pr is not from taipy...

Reply:
Yeah! ik, could you please guide how to fix it?